service/src/main/java/bio/terra/cbas/dependencies/common/DependencyUrlLoader.java

```diff
@@ -43,11 +43,9 @@ public String load(@NotNull DependencyUrlType key)
               throws DependencyNotAvailableException {
             if (key == DependencyUrlType.WDS_URL) {
               return fetchWdsUrl();
-            } else if (key == DependencyUrlType.CROMWELL_URL) {
+            } else {
```

Comment:
Yes, absolutely, we definitely need to change this cache to be specific to the user. We should also make sure that even if a user _does_ get given the wrong Cromwell client from the cache by accident, they still can't use it because they have to supply their token each time and they'd have the wrong token

Reply:
If it's okay with both of you, I'd like to do that as part of the next ticket since that adds "as the user" functionality to cbas.